spec/helpers/registry_spec_helper.rb

```diff
@@ -0,0 +1,53 @@
+module RegistrySpecHelper
+  def stub_people_registry_request
+    stub_request(:get, "http://search.dev.gov.uk/search.json?count=1500&fields%5B%5D=slug&fields%5B%5D=title&filter_format=person&start=0")
```

Comment:
pedantic point: you could use  ``` .with(query: {"a" => ["b", "c"]}``` to make this clearer? If you use ```hash_including``` you could even omit unimportant parameters

Reply:
Thanks a lot, that's much nicer. Changed.